bedrock/firefox/templates/firefox/features/send-tabs.html

```diff
@@ -0,0 +1,151 @@
+          </p>
+        </div>
+
+        <div class="show-fx-31-signed-out show-fx-android show-fx-ios">
+          <strong>{{ _('Log in to start sharing') }}</strong>
+          <p>
+            {{ _('Sign in to your Firefox account to send tabs between devices.') }}
+          </p>
+        </div>
+
+        <div class="show-not-fx show-default">
+          <p>{{ _('Get Firefox on your devices to get started. Then, sign into your Firefox Account to share tabs.') }}</p>
+        </div>
+      </div>{#--/#sync-intro-copy--#}
+
+      <div id="sync-intro-cta">
+        <div class="show-fx-31-signed-in">
+          <ul class="primary-buttons">
+            <li>
+              {{ google_play_button(id='cta-android', anchor_attributes={'data-download-location': 'other'}) }}
+            </li>
+            <li>
+              <a href="{{ firefox_ios_url('mozorg-sync_page-appstore-button') }}" data-link-type="download" data-download-os="iOS" data-download-location="other" id="cta-ios">
+                <img src="{{ l10n_img('firefox/ios/btn-app-store.svg') }}" alt="{{ _('Download on the App Store') }}" width="152" height="45">
+              </a>
+            </li>
+          </ul>
+
+          <p class="button-footer">
+            <a href="https://support.mozilla.org/kb/how-do-i-set-up-firefox-sync" class="ga-link" data-interaction="outbound link click">
+              {{ _('Need help?') }}
+            </a>
+          </p>
+        </div>
+
+        <div class="show-fx-31-signed-out">
+          <button class="button button-hollow" data-button-name="Get started with Sync" data-cta-position="Primary" id="cta-sync">
+            {{ _('Get started with Sync') }}
+          </button>
+        </div>
+
+        <div class="show-fx-30-older">
+          <a href="https://support.mozilla.org/kb/update-firefox-latest-version" class="button button-hollow" data-interaction="button click" data-element-action="Sync CTA" id="cta-update">
+            {{ _('Update your Firefox') }}
+          </a>
+        </div>
+
+        <div class="show-not-fx">
+          {{ download_firefox(button_color='button-arrow', download_location='other') }}
+        </div>
+      </div>{#--/#sync-intro-cta--#}
+    </div>{#--/#sync-intro-copy-cta-wrapper--#}
+  </div>{#--/.content--#}
+</section>
+
+<section class="features-list-section">
+  <div class="content">
+    <h2 class="section-title"><span>{{ _('Features') }}</span></h2>
+
+    <ul class="features-list">
+      <li class="features-list-item">
+        <img src="{{ static('img/firefox/features/thumbnails/send-tabs/still-emailing.jpg') }}" alt="">
+        <h3>{{ _('Free up your inbox') }}</h3>
+        <p>
+          {{ _('Still email links to yourself? Send tabs between devices instead.') }}
+        </p>
+      </li>
+
+      <li class="features-list-item">
+        <img src="{{ static('img/firefox/features/thumbnails/send-tabs/device-hop.jpg') }}" alt="">
+        <h3>{{ _('Device hop like a pro') }}</h3>
```

Comment:
I know `Device hop` can be a bit confusing for some translations. Would it be worth adding an l10n comment to provide an explanation?